storage/libradb/src/event_store/test.rs

```diff
@@ -160,24 +160,18 @@ fn traverse_events_by_access_path(
 }
 
 fn arb_event_batches() -> impl Strategy<Value = (Vec<AccessPath>, Vec<Vec<ContractEvent>>)> {
-    (
-        vec(any::<AccountAddress>(), 3),
-        hash_set(any::<Vec<u8>>(), 3),
-        (0..100usize),
-    )
-        .prop_flat_map(|(addresses, event_paths, num_batches)| {
-            let all_possible_access_paths = addresses
+    (vec(any::<AccountAddress>(), 3), (0..100usize))
+        .prop_flat_map(|(addresses, num_batches)| {
```

Comment:
Bleh, could you add a TODO here to remove this prop_flat_map? It isn't necessary.